SlevomatCodingStandard/Sniffs/Classes/SingleLineMethodDeclarationWhenPossibleSniff.php

```diff
@@ -0,0 +1,104 @@
+<?php declare(strict_types = 1);
+
+namespace SlevomatCodingStandard\Sniffs\Classes;
+
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\Sniff;
+use SlevomatCodingStandard\Helpers\FunctionHelper;
+use SlevomatCodingStandard\Helpers\TokenHelper;
+use function assert;
+use function in_array;
+use function is_bool;
+use function is_int;
+use function is_string;
+use function preg_replace;
+use function rtrim;
+use function sprintf;
+use function str_replace;
+use function strlen;
+use const T_FUNCTION;
+use const T_OPEN_CURLY_BRACKET;
+use const T_SEMICOLON;
+use const T_WHITESPACE;
+
+class SingleLineMethodDeclarationWhenPossibleSniff implements Sniff
+{
+
+	public const CODE_UNNECESSARY_MULTI_LINE_METHOD = 'UnnecessaryMultiLineMethod';
+
+	/** @var int */
+	public $maxLineLength = 120;
```

Comment:
Will `0` work for all methods?

Reply:
Not sure if I understand, but ... if you set this to 0, then this sniff effectively does nothing, because all methods will be considered as not inlinable. If you set this to lets say 10000, all methods will be required to have an inline definition.

Reply:
@grongor Ok, can we support `null` (or probably empty string because of PHPCS configuration) to require all methods to be on one line?

Reply:
Ok, what if I adjusted the code to use 0 value for that? Because having it "do nothing" doesn't make sense and it's just a side-effect.

Reply:
Ok :)